articles/redis-cache/cache-overview.md

```diff
@@ -76,9 +76,11 @@ The [Redis Cache Pricing](https://azure.microsoft.com/pricing/details/cache/) pa
   Create a simple ASP.NET web app that uses an Azure Redis Cache.
 * [.NET Quickstart](cache-dotnet-how-to-use-azure-redis-cache.md)
   Create a .NET app that uses an Azure Redis Cache.
+* [.NET Core Quickstart](cache-dotnet-core-quickstart.md)
+  Create a .NET app that uses an Azure Redis Cache.
```

Comment:
Description should also mention .NET Core app:

Create a .NET Core app that uses an Azure Redis Cache.